src/mixer.erl

```diff
@@ -105,6 +107,16 @@ expand_mixin(Line, Name) when is_atom(Name) ->
             [#mixin{line=Line, mod=Name, fname=Fun, alias=Fun, arity=Arity} || {Fun, Arity} <- Exports,
                                                                                Fun /= module_info]
```

Comment:
Line 108 is too long:                                                                                Fun /= module_info].